pkg/utils/unit.go

```diff
@@ -8,3 +17,33 @@ const (
 	GB
 	TB
 )
+
+type logCollector struct {
+	mu     sync.Mutex
+	fields []zap.Field
+}
+
+// LogCollector collects time into summary log
+var LogCollector *logCollector
```

Comment:
We should provide an API to **replace** the default collector which is similar to
 https://godoc.org/google.golang.org/grpc/grpclog#SetLoggerV2

Reply:
[Done](https://github.com/pingcap/br/pull/118/files#diff-500e3c8287bac6a5d41b94b101065518R52)